In Ubuntu, we have only opencv4 available, and the name
of the pkg-config file is qualified with the version number.

Index: b/zart/zart.pro
===================================================================
--- a/zart/zart.pro
+++ b/zart/zart.pro
@@ -29,7 +29,7 @@ greaterThan(QT_MAJOR_VERSION, 4): CONFIG
 CONFIG	+= warn_on
 QT_CONFIG -= no-pkg-config
 CONFIG += link_pkgconfig
-PKGCONFIG += opencv fftw3 zlib
+PKGCONFIG += opencv4 fftw3 zlib
 DEFINES += cimg_use_fftw3 cimg_use_zlib
 
 DEFINES += QT_DEPRECATED_WARNINGS
Index: b/zart/include/ImageConverter.h
===================================================================
--- a/zart/include/ImageConverter.h
+++ b/zart/include/ImageConverter.h
@@ -48,6 +48,7 @@
 
 #include <QMutex>
 #include <opencv2/opencv.hpp>
+#include <opencv2/imgproc/imgproc_c.h>
 #ifndef gmic_build
 #include "CImg.h"
 #endif
